apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.ts

```diff
@@ -31,6 +32,15 @@ function assertCanCreateQuestion(resLocals: Record<string, any>) {
   }
 }
 
+router.use(
+  asyncHandler(async (req, res, next) => {
+    if (!(await features.enabledFromLocals('ai-question-generation', res.locals))) {
+      throw new error.HttpStatusError(403, 'Feature not enabled');
+    }
+    next();
+  }),
+);
```

Comment:
This is copied from here:

https://github.com/PrairieLearn/PrairieLearn/blob/df6853eaa2bf4d0ef6024d397ac19a515706c891/apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.ts#L149-L156